waiter/src/waiter/token.clj

```diff
@@ -499,7 +512,9 @@
                              (fn [[token entry]]
                                (cond-> (assoc entry :owner owner :token token)
                                  (not show-metadata)
-                                 (dissoc :deleted :etag)))))))
+                                 (dissoc :deleted :etag :last-update-time)
+                                 show-metadata
+                                 (update :last-update-time tc/from-long)))))))
```

Comment:
I think `cond->` here is odd, since the conditions are `x` and `not x`.

```clojure
(->
  (if show-metadata
    (update entry :last-update-time tc/from-long)
    (dissoc entry :deleted :etag :last-update-time))
  (assoc :owner owner :token token))
```

Reply:
Done.